packages/web-service/src/pages/habitat/a24/sett/common/date/dates.js

```diff
@@ -0,0 +1,24 @@
+import { APIRequests } from '../../../../../../services/api-requests.js'
+import { A24_SETT } from '../../../../../tasklist/a24-badger-licence.js'
+import { isCompleteOrConfirmed } from '../../../../../common/tag-functions.js'
+import { habitatURIs } from '../../../../../../uris.js'
+
+export const getDateData = (dateToGet) => {
+  if (dateToGet) {
```

Comment:
fixes - sonar cloud duplicate issue

and a bug where the endDate session is failing.